features/gamepad.yml

```
@@ -0,0 +1,25 @@
+name: Gamepad
+description: The `Gamepad` API accesses and responds to signals from game controllers.
```

Small suggestion here to show the common entrypoint to the gamepad data itself:

```suggestion
description: The `navigator.getGamepads()` API accesses and responds to signals from game controllers.
```